src/ploomber_core/warnings.py

```diff
@@ -6,4 +11,15 @@ class PloomberDeprecationWarning(FutureWarning):
         Changed superclass from UserWarning to FutureWarning
     """
 
-    pass
+    # @
+    def __init__(self, message):
+        print("message: ", message)
+        self.message = message
+
+
+def deprecation_warning(message):
```

Comment:
we already have some abstractions in the telemetry module, you can use those instead of calling posthog.capture

also, you can set an env var to send events to a different posthog project for debugging: https://github.com/ploomber/core/blob/ed34d76898ca792c9ba8cfc691b7c1b078b13d80/src/ploomber_core/telemetry/telemetry.py#L295